src/views/EventInitiation.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="page-container">
+    <NavBtns
+      :topType="'share-dark'"
+      :botType="currStep === 0 ? 'next-only' : 'default'"
+      @next="nextStep()"
+      @back="prevStep()"
+    />
+
+    <!-- step 1 -->
+    <div class="step-1 step" v-if="currStep === 0">
+      <img src="../assets/icons/event-initiation/hourglass-icon.svg" alt class="step-icon" />
+      <h1 class="step-title">Date / Time</h1>
+      <div class="date-wrapper">
+        <v-menu
+          ref="calendarMenu"
+          max-width="290"
+          :close-on-content-click="false"
+          offset-y
+          :return-value.sync="eventDate"
+        >
+          <template v-slot:activator="{ on }">
+            <textarea
+              type="text"
+              v-on="on"
+              placeholder="When should the event be held?"
+              class="step-input"
+              v-model="eventDate"
+            ></textarea>
+          </template>
+          <v-date-picker v-model="eventDate">
+            <v-btn text color="primary" @click="showCalendar = false">Cancel</v-btn>
```

Comment:
Somehow cancel button doesn't work / close the date-picker

Reply:
i'll look into this